.github/workflows/docker-image-build-push.yml

```diff
@@ -95,4 +103,55 @@ jobs:
       - name: Clean up environment
         if: always()
         run: |
-          rm -f ${HOME}/.docker/config.json
\ No newline at end of file
+          rm -f ${HOME}/.docker/config.json
+  
+  dev_repository_dispatch:
```

Comment:
This workflow was originally meant just to `Build and publish Docker image`. If we're thinking of deploying, perhaps we should create a separate workflow for that, maybe using the `workflow_call` feature to reuse building docker logic. This approach might help us keep things clearer and more organized.